packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -54,18 +57,23 @@
       "pattern": "^[a-z]{1,}(-[a-zA-Z0-9]{1,})*$"
     },
     "country": {
+      "description": "The country in which the customer is located.",
```

Comment:
Could you add a bit more detail on the format of this field, where it's an ISO code or the country name? AFAIK from talking to Bianca its country name, translated based on locale, but I'm not 100% sure.

Reply:
Thanks for the clarification! I'll make this update.